ote/src/cljs/ote/ui/mui_chip_input.cljs

```diff
@@ -47,8 +47,15 @@
                    (if @auto-select?
```

Comment:
Can we make a drive-by fix and remove the auto-select atom? I don't see the need for it. It is just reset! on render. If it is needed, for some reason, document it.